Makefile

```diff
@@ -71,10 +71,10 @@ db-build: trivy-db
 
 .PHONY: db-compact
 db-compact: $(GOBIN)/bbolt cache/db/trivy.db
-	mkdir assets/
+	mkdir -p assets/
```

Comment:
single level directory structure is created. Why is `-p`  required?

Reply:
I just wanted to avoid failing the command when the directory exists.

```
$ mkdir assets
$ mkdir assets
mkdir: cannot create directory ‘assets’: File exists
```

```
$ mkdir -p assets
$ mkdir -p assets
```